src/template.rs

```diff
@@ -134,13 +145,21 @@ fn to_source(pkg: &ResolvedPackage<'_>, cwd: &Path) -> Result<Source> {
                 .ok_or(anyhow!("checksum is required for crates.io package {}", id))?,
         }
     } else if id.source_id().is_git() {
+        let branch = if let Some(cargo::core::source::GitReference::Branch(branch)) =
```

Comment:
Minor improvement idea,

why not add `source::GitReference` to `use cargo::core::{...}`

then change this to GitReference::Branch?